fooof/synth/transform.py

```diff
@@ -14,33 +16,124 @@ def rotate_spectrum(freqs, power_spectrum, delta_f, f_rotation):
         Frequency axis of input power spectrum, in Hz.
     power_spectrum : 1d array
         Power values of the spectrum that is to be rotated.
-    delta_f : float
-        Change in power law exponent to be applied.
-        Positive is counterclockwise rotation (flatten).
-        Negative is clockwise rotation (steepen).
+    delta : float
```

Comment:
maybe we should name it delta_chi or delta_exp or something? delta by itself is kinda generic